File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.java
import java.util.*;
class Solution {
    public int solution(String myString, String pat) {
		int answer = 0;

        for(int i = 0; i < myString.length(); i++) {
        	int count = 0;

        	if(i + pat.length() > myString.length()) break;
        	
        	for(int j = 0; j < pat.length(); j++) {
        		if(myString.charAt(i+j) == pat.charAt(j)) {
        			count++;
        		}else {
        			count = 0;
        			break;
        		}
        	}
        	
        	if(count == pat.length()) answer++;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.py
def solution(myString, pat):
    answer = 0
    
    # 1)
    # while True:
    #     if myString.find(pat) > -1:
    #         answer += 1
    #         myString = myString[myString.find(pat) + 1:]
    #     else :
    #         break;
    
    for i in range(len(myString)):
        if myString[i:len(pat) + i] == pat:
            answer += 1
    
    return answer